Data/Sys/GameSettings/GEZE8P.ini

```diff
@@ -0,0 +1,42 @@
+# GEZE8P - BillyHatcher
```

Comment:
Maybe not for this PR but more and more the old INIs have names that are bugging me.